app/src/main/java/it/quip/android/partials/EndlessScrollHandler.java

```diff
@@ -0,0 +1,47 @@
+package it.quip.android.partials;
```

Comment:
@hashamali packages have singular naming. eg. model, activity, etc. Let's call this partial